concepts/conceptpower.py

```diff
@@ -49,10 +52,18 @@ def get(self, uri, headers):
             raise ValueError(f"Error fetching concept data: {response.status_code}")
         return concept_entries[0] if concept_entries else {}
 
-    def create(self, user, password, label, pos, conceptlist, description,
+    def create(self, user, label, pos, conceptlist, description,
```

Comment:
just as important as throwing an exception when conceptpower credentials are missing is to throw one when the authentication is incorrect. right now it looks like it would just show a generic 'something went wrong' message to the user.